person.rb

```diff
@@ -1,8 +1,13 @@
-class Person
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
+  def correct_name
+    @name = name
+  end
+
```

Comment:
- [ ] you're already assigning the name to the instance variable @name in the initialize method, it would be better to put this method below the initialize method and only make this method look like below:
```
def correct_name
  name
end
```